packages/sui-analytics/src/index.js

```diff
@@ -12,7 +14,17 @@ export default {
         anonymousId: window.analytics.user().anonymousId(),
         userId: window.analytics.user().id()
       }
+      if (window && window.Visitor && adobeOrgId) {
+        options['Adobe Analytics'] = {
+          marketingCloudVisitorId: window.Visitor.getInstance(
```

Comment:
I think that maybe this is too much business-related to being here. Maybe we can develop the concept of middleware inside of sui-analytic